### PR TITLE
updated kin.js example

### PR DESCRIPTION
**Pull request by vbchen**
Updated the example kinemage.
Updated the kin.js code with the additions to kin-parser.

### CODE REVIEW

File: examples/scripts/parser/kin.js [hunk 1]
```diff
@@ -1,42 +1,62 @@
...
-      if (!dotList.master.includes(master)) return
-      for (var i = 0, il = dotList.position.length / 3; i < il; ++i) {
-        var i3 = i * 3
-        var x = dotList.position[ i3 ]
-        var y = dotList.position[ i3 + 1 ]
-        var z = dotList.position[ i3 + 2 ]
-        var r = dotList.color[ i3 ]
-        var g = dotList.color[ i3 + 1 ]
-        var b = dotList.color[ i3 + 2 ]
-        shape.addPoint([ x, y, z ], [ r, g, b ], dotList.label[ i ])
```
Comment (arose) [inline examples/scripts/parser/kin.js]: You should actually use the shape approach here to get support for picking/hover labels

File: examples/scripts/parser/kin.js [hunk 2]
```diff
@@ -1,42 +1,62 @@
...
+    })
+
+    var positionArray = []
+    var colorArray = []
+    kinemage.ribbonLists.forEach(function (ribbonList) {
+      if (!ribbonList.masterArray.includes(master)) return
+      positionArray.push(...ribbonList.positionArray)
+      colorArray.push(...ribbonList.colorArray)
     })
+    if (positionArray.length > 0) {
```
Comment (arose) [inline examples/scripts/parser/kin.js]: For now it is ok to use a buffer here but we should add a method `shape.addTriangle` to the `Shape` class to get support for picking/hover labels here as well.

Comment (arose) [reply examples/scripts/parser/kin.js]: Here is the commit (https://github.com/arose/ngl/commit/545cc9387777205acdb7cf71b8e3f430816ad421) where I added `shape.addPoint` and `shape.addWideline` if you want to give it a try.

File: examples/scripts/parser/kin.js (hunk 1 above)
Comment (vbchen) [reply examples/scripts/parser/kin.js]: Ah, ok, cool!  I'll give it a try.  That's my fault for not looking more closely at your recent commits.

File: examples/scripts/parser/kin.js (hunk 2 above)
Comment (vbchen) [reply examples/scripts/parser/kin.js]: Sounds good, I'll give it a go.

Comment (arose) [reply examples/scripts/parser/kin.js]: great, thanks!

**Pull request closed without merging**